packages/vm/tests/tester/config.ts

```diff
@@ -258,7 +258,12 @@ export function getTestDirs(network: string, testType: string) {
  * @param {String} network - the network field of a test
  * @returns {Common} the Common which should be used
  */
-export function getCommon(network: string) {
+export function getCommon(targetNetwork: string) {
+  let network = targetNetwork
+  if (network.includes('+')) {
```

Comment:
could you include some explanation comment here what the `+` symbol represents? (or maybe can include in the typedoc)